packages/libs/react-ui/src/components/Form/Input/Input.stories.tsx

```diff
@@ -87,17 +86,6 @@ const meta: Meta<IInputProps> = {
         type: 'text',
       },
     },
-    leadingTextWidth: {
-      description:
-        'Width of the leading text. Defaults to the size of the text itself.',
-      control: {
-        type: 'select',
-      },
-      options: [
-        '- Omit this property to auto-size the leading text',
-        ...Object.keys(vars.sizes).map((key) => key as keyof typeof vars.sizes),
-      ],
-    },
```

Comment:
Why did you remove this? This should still be possible to apply via the Input

Reply:
I thought this always came throught the wrapper, not as a prop as well, i can bring it back